### PR TITLE
[docs] fix workflow example

### PR DESCRIPTION
**Pull request by Comradin**
The directory for the checkout must be created before we can use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ At a high level an operator using the SDK processes events for watched resources
 First, checkout and install the operator-sdk CLI:
 
 ```sh
+$ mkdir -p $GOPATH/src/github.com/operator-framework/operator-sdk
 $ cd $GOPATH/src/github.com/operator-framework/operator-sdk
```
Comment (hasbro17) [inline README.md]: @Comradin I think what's confusing is the missing `git clone` step. We can add that in to be more explicit.
So can you please replace lines 38-39 with:
```sh
$ mkdir -p $GOPATH/src/github.com/operator-framework
$ cd $GOPATH/src/github.com/operator-framework
$ git clone github.com/operator-framework/operator-sdk
$ cd operator-sdk
```

**Comment by Comradin** [comment]
@hasbro17 Thanks for your propose, add the lines.

Sorry for the delay, did not check my github

**Comment by fanminshi** [comment]
@Comradin could you also update the [user_guide](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#install-the-operator-sdk-cli) to make installing the operator-sdk more explicit?

**Comment by Comradin** [comment]
@fanminshi Thanks for the suggest, adjusted the file, too

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**